lightwood/encoders/time_series/rnn.py

```diff
@@ -5,122 +5,171 @@
 from lightwood.helpers.device import get_devices
 from lightwood.encoders.encoder_base import EncoderBase
 
+import numpy as np
 import torch
 import torch.nn as nn
 from torch import optim
-import numpy as np
 
 
 class RnnEncoder(EncoderBase):
+    def __init__(self, encoded_vector_size=4, train_iters=75000, stop_on_error=0.8, learning_rate=0.01,
+                 is_target=False, ts_n_dims=1, dropout=0.2, weight_decay=1e-4, max_timesteps=64):
```

Comment:
The interface here is *probably* too complex, in that, it will probably never be used in full, it might make more sense to just set these hyperparameters internally and move them to the interface once an external module needs to actually modify them.